src/css/index.scss

```diff
@@ -0,0 +1,16 @@
+.main-display-wrapper {
+  width: 800px;
+  margin: 60px auto;
+  .date-banner {
+    margin: 18px 0px;
+  }
+  #date-banner-data {
+    margin-left: 8px;
+  }
+}
+.loading-mask {
```

Comment:
is this `loading-mask` have any styles?
If not please remove.

But it is good for you to notice the need to add a loading spinner.